app/controllers/order_notes_controller.rb

```diff
@@ -0,0 +1,9 @@
+class OrderNotesController < ApplicationController
+  def destroy
+    order_note = OrderNote.find(params[:id])
+    order_note.destroy
+
+    flash[:success] = "Order note: #{order_note.text} for order #{order_note.order_id} deleted!"
```

Comment:
If the note text is expected to be somewhat large, this might be a messy looking notification message. If you are ok with that, then :+1: